infra/modules/github_selfhosted_runner_on_container_app_jobs/main.tf

```diff
@@ -1,35 +1,22 @@
 terraform {
+
   required_providers {
     azurerm = {
       source  = "hashicorp/azurerm"
-      version = "~>3.86"
+      version = ">= 3.110, < 5.0"
```

Comment:
does this works even for v3 codebases or needs a provider upgrade? in other workds: is this a breaking change?

Reply:
No, as clients can (and do) manage version on their side setting the preferred one